webpki-ccadb/src/lib.rs

```diff
@@ -102,23 +103,37 @@ pub struct CertificateMetadata {
 }
 
 impl CertificateMetadata {
-    /// Returns true iff the certificate has valid TrustBits that include TrustBits::Websites,
-    /// and the certificate has no distrust for TLS after date. In all other cases this function
-    /// returns false.
+    /// Returns true iff the issuer certificate should be considered trusted to issue TLS
```

Comment:
nit: preexisting, but I think we should expand "iff" -> "if and only if" to minimise jargon

Reply:
Good call. Fixed.